packages/i18n/src/locales/en_US.ts

```diff
@@ -166,6 +165,10 @@ const translations: Catalog = {
           description: 'Shell Help',
           link: 'https://docs.mongodb.com/manual/reference/method',
           attributes: {
+            log: {
+              description:
+                "'log.info(<msg>)': Writes a custom info/warn/error/fatal/debug message to the log file",
```

Comment:
```suggestion
                "'log.info(<msg>)': Write a custom info/warn/error/fatal/debug message to the log file",
```

The `show` entries also all use imperative